gumby/experiment.py

```diff
@@ -280,7 +280,8 @@ def get_changed_values(prev_dict, cur_dict):
 
         new_values, changed_values = get_changed_values(prev_dict, cur_dict)
         if changed_values:
-            self._stats_file.write('%.1f %s %s %s\n' % (time(), self.my_id, name, json.dumps(changed_values)))
```

Comment:
**Remark:** This stuff wasn't even working apparently, good find 😁. We could consider adding a unit test for it?

Reply:
It's a good idea, but I think it is better to do it in a separate PR, as currently, we don't have a suitable unit test file for such a test. We can potentially test a lot of staff in `gumby/experiment.py` and other files, so adding proper unit tests looks like a solid chunk of work.

Reply:
👍